.circleci/config.yml

```diff
@@ -424,7 +435,9 @@ workflows:
                     - build-windows
                 filters:
                     branches:
-                        only: main
+                        only:
+                          - main
```

Comment:
I believe we will want to make a prerelease only on `develop` and a real release on `main`. But that's not very important